app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,24 @@
+require 'net/http'
+
+class VotesController < WelcomeController
+  def create
+    @vote = Vote.new
+    @vote.date = Date.today.beginning_of_day
```

Comment:
I would store this as a time - so `voted_at` - perhaps you are using it to group by day - but SQL is also capable to grouping by day using a time rather than a date.

You could potentially make use of the `created_at` timestamp available, but in this case, I quite like having a domain field for when the vote was cast rather than relying on the created_at/updated_at timestamps as these can be reserved for system diagnostics rather than business function (IMO).

Reply:
`.date` or now `voted_at` is a `datetime` data types on Rails database side. But to simplify the requirements we fake the field as a date for now. Dealing with times is a bit of a pain to start with and thought should be too detailed for MVP. You have to jungle with SQL, timezones, times and the date of a time and validations.

We could to that into a separate story with more detailed requirements on when a batch can be added. Like:
* More than one a day. 
* What is the admin allowed to edit: Complete time, only date and hours? no need to be accurate on the minute or seconds?